quidem-ut/src/test/quidem/org.apache.druid.quidem.QTest/qaJsonCols/funcs_and_sql_func_json_value.03.dart.iq

```diff
@@ -533,14 +533,14 @@ FROM test_json_cols;
 #-------------------------------------------------------------------------
 SELECT json_value(c1, '$.a_bool' RETURNING boolean) c
 FROM test_json_cols;
-+-------+
```

Comment:
The changes in this file look like improvements to me, since the values in the file are actually boolean `true`.